aquia-inc/template-vite-react
# build(dev-deps): upgrade minor versions

## Summary

Upgraded several devDependencies to their latest versions to ensure compatibility with the latest features, improvements, and bug fixes.

### Added

- None

### Changed

- Upgraded `@testing-library/jest-dom` from `^6.4.2` to `^6.4.8`
- Upgraded `@testing-library/react` from `^14.2.1` to `^14.3.1`
- Upgraded `dotenv` from `^16.4.4` to `^16.4.5`
- Upgraded `jest-date-mock` from `^1.0.8` to `^1.0.10`
- Upgraded `rimraf` from `^5.0.5` to `^6.0.1`
- Upgraded `sass` from `^1.70.0` to `^1.77.8`

### Deprecated

- None

### Removed

- None

### Fixed

- None

## How to test

1. Checkout this branch: `git checkout origin/build/dependencies/7-dev-deps`
2. Run `yarn` and ensure all dependencies are correctly installed without duplication and that there are no issues related to missing or incompatible dependencies.
3. Run `yarn test` and verify that all tests run and pass.
4. Run `yarn dev` and verify that the application runs as expected.
5. Run `yarn build` and verify that the application builds as expected.
6. Run `yarn build-storybook` and verify the Storybook build succeeds.
